src/components/layout/nav/nav-desktop.tsx

```diff
@@ -0,0 +1,239 @@
+        setActiveDropdown(dropdown)
+        if (!target) return
+        setActiveLinkRef(target)
+    }
+
+    const setDropdownRef = (new_ref) => setActiveDropdownRef(new_ref)
+
+    useOutsideClick(navigation_bar_ref, () => setActiveDropdown(''), active_dropdown_ref)
+
+    useEffect(() => {
+        setMounted(true)
+        document.addEventListener('scroll', buttonHandleScroll, { passive: true })
+        return () => document.removeEventListener('scroll', buttonHandleScroll)
+    }, [])
+
+    const tabs = [
+        {
+            id: 1,
+            title: 'Trade',
+            active: checkActive('trade'),
+            onClick: (e) => handleLinkClick('trade', e.target),
+        },
+        {
+            id: 2,
+            title: 'Markets',
+            active: checkActive('markets'),
+            onClick: (e) => handleLinkClick('markets', e.target),
+        },
+        {
+            id: 3,
+            title: 'About us',
+            active: checkActive('about'),
+            onClick: (e) => handleLinkClick('about', e.target),
+        },
+        {
+            id: 4,
+            title: 'Resources',
+            active: checkActive('resources'),
+            onClick: (e) => handleLinkClick('resources', e.target),
+        },
+    ]
+
+    return (
+        <>
+            {active_dropdown && (
+                <PlatformsDropdown
+                    active_dropdown={active_dropdown}
+                    key={active_dropdown}
+                    current_ref={active_link_ref}
+                    parent={active_dropdown}
+                    is_ppc={is_ppc}
+                    is_ppc_redirect={is_ppc_redirect}
+                    setActiveDropdown={setDropdownRef}
+                />
+            )}
+
+            <Wrapper>
+                <NavLeftMain>
+                    <LogoLink
+                        to={!is_ppc_redirect ? base || '/' : '/landing'}
+                        aria-label={localize('Home')}
+                    >
+                        <QueryImage
+                            data={data['deriv']}
+                            alt={localize('Deriv')}
+                            max_width="16.4rem"
+                            width="100%"
+                            height="auto"
+                        />
+                    </LogoLink>
+                    <Line />
+                    {academy_logo ? (
+                        <img src={AcademyLogo} alt={'Academy'} />
+                    ) : (
+                        <img src={LogoCombinedShape} alt={'logo combined shape'} />
+                    )}
+                </NavLeftMain>
+
+                <NavCenter ref={navigation_bar_ref}>
+                    {tabs.map((item) => {
+                        return (
+                            <NavLink key={`tab-${item.id}`} onClick={item.onClick}>
+                                <StyledButton
+                                    aria-label={localize(item.title)}
```

Comment:
we can't pass variables to localize() since we have file readers for translation generator.